lib/travis/api/v3/renderer/log.rb

```diff
@@ -1,10 +1,28 @@
+require 'travis/api/v3/log_token'
+
 module Travis::API::V3
   class Renderer::Log < ModelRenderer
     def self.render(model, representation = :standard, **options)
       return super unless options[:accept] == 'text/plain'.freeze
       model.content
     end
 
+    def render(representation)
+      result = super
+
+      raw_log_href = "#{href}.txt"
+      if raw_log_href !~ /^\/v3/
```

Comment:
Looks like this introduced a bug in Enterprise as the api lives at `/api` instead of `api.[hostname]`. As a result this always returns as true and you get URLs like: `/v3/api/v3/job/61/log.txt` instead of `/api/v3/job/61/log.txt`

But the fact that `v3` is in there already makes me wonder if this is still needed?

Reply:
ah, seeing some ` "@raw_log_href": "/v3/api/job/221/log.txt"` as well, so that code may be needed after all. 

This should be  `"@raw_log_href": "/api/v3/job/221/log.txt"`

Reply:
PR to address this here: https://github.com/travis-ci/travis-api/pull/692